Makefile

```diff
@@ -1,18 +1,36 @@
-BUILD_NUMBER := 1
-CP_VERSION := 3.3.0-SNAPSHOT
+# You can override vars like REPOSITORY in a local.make file
+-include local.make
+
+# Bump this on subsequent build, reset on new version or public release. Inherit from env for CI builds.
+BUILD_NUMBER ?= 1
+
+CONFLUENT_MAJOR_VERSION ?= 3
+CONFLUENT_MINOR_VERSION ?= 3
+CONFLUENT_PATCH_VERSION ?= 0
+
+CONFLUENT_VERSION ?= ${CONFLUENT_MAJOR_VERSION}.${CONFLUENT_MINOR_VERSION}.${CONFLUENT_PATCH_VERSION}
```

Comment:
You're right that it should be in the label. I've updated this PR to do so.

For other cases, SNAPSHOT will be sent as the value of CONFLUENT_MVN_LABEL, empty for non-CI builds (but usable for special occasions if we need it)